includes/admin/class-amp-editor-blocks.php

```diff
@@ -120,49 +119,6 @@ public function whitelist_block_atts_in_wp_kses_allowed_html( $tags, $context )
 		return $tags;
 	}
 
-	/**
-	 * Enqueue filters for extending core blocks attributes.
-	 * Has to be loaded before registering the blocks in registerCoreBlocks.
-	 */
-	public function enqueue_block_editor_assets() {
-		$script_handle = 'amp-editor-blocks';
-
-		if ( AMP_Story_Post_Type::POST_TYPE_SLUG !== get_current_screen()->post_type ) {
-			wp_enqueue_style(
-				'amp-editor-blocks-style',
-				amp_get_asset_url( 'css/amp-editor-blocks.css' ),
-				array(),
-				AMP__VERSION
-			);
-		}
-
-		$script_deps_path    = AMP__DIR__ . '/assets/js/amp-editor-blocks.deps.json';
-		$script_dependencies = file_exists( $script_deps_path )
-			? json_decode( file_get_contents( $script_deps_path ), false ) // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
-			: array();
-
-		wp_enqueue_script(
-			$script_handle,
-			amp_get_asset_url( 'js/amp-editor-blocks.js' ),
-			$script_dependencies,
-			AMP__VERSION,
-			true
-		);
-
-		if ( function_exists( 'wp_set_script_translations' ) ) {
-			wp_set_script_translations( $script_handle, 'amp' );
```

Comment:
Curious to know -- is `wp_set_script_translations` not needed anymore?

Reply:
This `wp_set_script_translations` call here is for `amp-editor-blocks.js`, which this PR removes 🙂 

`wp_set_script_translations` for the one and only `amp-block-editor.js` script that's left is still in `AMP_Post_Meta_Box`.

Reply:
Ah, makes sense, thanks!